docs/extending.rst

```diff
@@ -44,6 +44,35 @@ A more elegant way can be to wrap ``attrs`` altogether and build a class `DSL <h
 
 An example for that is the package `environ-config <https://github.com/hynek/environ-config>`_ that uses ``attrs`` under the hood to define environment-based configurations declaratively without exposing ``attrs`` APIs at all.
 
+Another common use case is to overwrite ``attrs``'s defaults.
```

Comment:
I worry about putting this line right next to the mypy stuff because there's no way change a default (e.g. `eq` or `order`.) The only thing the plugin lets you do is add a method that has the same defaults as three methods you're overriding.  But maybe it won't be a problem.

Reply:
Agreed, that should be clarified. :|